Makefile

```diff
@@ -240,11 +240,11 @@ test:
 ### license-check:    Check Lua source code for Apache License
 .PHONY: license-check
 license-check:
-ifeq ("$(wildcard .travis/openwhisk-utilities/scancode/scanCode.py)", "")
-	git clone https://github.com/apache/openwhisk-utilities.git .travis/openwhisk-utilities
-	cp .travis/ASF* .travis/openwhisk-utilities/scancode/
+ifeq ("$(wildcard ci/openwhisk-utilities/scancode/scanCode.py)", "")
+	git clone https://github.com/apache/openwhisk-utilities.git ci/openwhisk-utilities
+	cp ci/ASF* ci/openwhisk-utilities/scancode/
```

Comment:
I see SkyWalking has one Action called https://github.com/apache/skywalking-eyes , we may have a try haha

Reply:
wow that's the idea coming from the podcast of ACL Beijing discussing summer 2021 and they really made it! Cool!